ftplugin/cs/sharpenup.vim

```diff
@@ -68,4 +68,9 @@ if g:sharpenup_codeactions
   let b:undo_ftplugin .= '| execute "autocmd! sharpenup_ftplugin * <buffer>"'
 endif
 
+if g:sharpenup_legacy_csproj_actions
```

Comment:
This section isn't quite right. The mappings should use the same "prefix" as the other sharpenup mappings, which is assigned to `s:pre` on line 16. It's inside an `if` block, and this section should be moved inside that block (these mappings should not be created if `g:sharpenup_create_mappings == 0`.

To create a "dynamic" mapping you'll need to put it in an `execute` command, so it will look like this:

```vim
execute 'nmap <silent> <buffer> ' . s:pre . 'add :call sharpenup#legacycsproj#AddToProject()<CR>'
```